src/currencyEngineETH.js

```diff
@@ -889,6 +896,54 @@ class EthereumEngine {
       this.log('Error fetching networkFees:')
       this.log(err)
     }
+
+    try {
+      const url = sprintf('https://www.ethgasstation.info/json/ethgasAPI.json')
```

Comment:
Can you move this URI string into the currencyInfo.defaultSettings.feeInfoServer like in the bitcoin plugin for constancy: 
https://github.com/Airbitz/edge-currency-bitcoin/blob/master/src/info/bitcoin.js

Reply:
Yes I could except this is a very specific server which I don't expect to be able to change without breaking the API, unlike the info1 server which we may have several of. So for now I'd like to keep it simple.

Reply:
I'm not talking about the infoServer.
In bitcoin we have "feeInfoServer" and "infoServer"
feeInfoServer is the "bitcoinfees.21" uri.
I think it's just my OCD of trying to avoid any strings/numbers in the code.
It's more of a style thing, it's not super important if you don't feel like doing it.